setup.cfg

```diff
@@ -43,7 +43,7 @@ install_requires =
     pluggy >= 0.12.0, <1
     py >= 1.4.17, <2
     six >= 1.14.0, <2  # required when virtualenv>=20
-    virtualenv >= 16.0.0
+    virtualenv >= 16.0.0, !=20.0.0, !=20.0.1, !=20.0.2, !=20.0.3, !=20.0.4, !=20.0.5, !=20.0.6, !=20.0.7
```

Comment:
Why is this needed? Shouldn't install just pick the latest?

Reply:
additionally, though each of these have bugs they "work" for the 95-99% case and it's probably not helpful to exclude it for those already using those versions

Reply:
It'll only install latest if no existing virtualenv is present. So if you do `pip install tox` and there's `virtualenv==20.0.2` there, it'll not upgrade it because it matches the requirement.

Reply:
I can't think of a better solution atm

Reply:
😢 you're right